vault/core_util.go

```diff
@@ -213,3 +214,9 @@ func DiagnoseCheckLicense(ctx context.Context, vaultCore *Core, coreConfig CoreC
 func createCustomMessageManager(storage logical.Storage, _ *Core) CustomMessagesManager {
 	return uicustommessages.NewManager(storage)
 }
+
+func (c *Core) GetRequestLimiter(key string) *limits.RequestLimiter {
```

Comment:
Nit: should we add a doc comment about what this is and that callers must expect to get nil returned in some cases (e.g. no limiter, CE)?